gyp/pylib/gyp/common.py

```diff
@@ -429,6 +429,10 @@ def GetFlavor(params):
     return 'netbsd'
   if sys.platform.startswith('aix'):
     return 'aix'
+  if sys.platform.startswith('zos'):
+    return 'zos'
+  if sys.platform.startswith('os390'):
+    return 'zos'
```

Comment:
https://docs.python.org/3/library/stdtypes.html#str.startswith Says __prefix can also be a tuple of prefixes to look for.__ so:
```python
if sys.platform.startswith(('os390', 'zos')):
    return 'zos'
```